### PR TITLE
Conform to our clang-format

### PR DESCRIPTION
**Pull request by kylef**
This PR conforms to the clang-format configuration that we previously agreed upon.

```
clang-format version 5.0.0 (tags/google/stable/2017-06-22)
```

### CODE REVIEW

**Comment by kylef** [comment]
Does seem that our clang-format is breaking the syntax in some area (CI is failing).

**Comment by w-vi** [comment]
@kylef I guess that it needs some little tweaking, have you tried it locally?

**Comment by tjanc** [comment]
Copied the .clang-format from snowcrash/master and applied it after a rebase.

**Comment by w-vi** [comment]
:clap:

File: .clang-format [hunk 1]
```diff
@@ -0,0 +1,74 @@
+Language:        Cpp
```
Comment (kylef) [inline .clang-format]: We could probably DRY this up and make it a symbolic reference to the clang format in the sundown sub module.

Comment (tjanc) [inline .clang-format]: How will non-unix systems deal with that?

Comment (kylef) [inline .clang-format]: NTFS has symbolic links so I'd expect it to work on Windows too, but not 100% sure.

**Pull request merged**